aip/general/0162.md

```diff
@@ -379,6 +379,9 @@ message DeleteBookRevisionRequest {
   (this could lead to dangerous or confusing mistakes).
 - If the resource supports soft delete, then revisions of that resource
   **should** also support soft delete.
+- The API **must not** cause the _resource_ to be deleted.
+  - If the resource cannot exist with no revisions, then the API **must** fail 
+    with a `FAILED_PRECONDITION` error if the request targets the only revision.
```

Comment:
I wonder if we should be more prescriptive here, e.g. say that:

```suggestion
Because a resource **should not** exist with zero revisions, the method **must** fail
with `FAILED_PRECONDITION` if the user attempts to delete the only revision.
```

Reply:
Adopted this suggestion